cmd/worker/main.go

```diff
@@ -185,11 +211,16 @@ func main() {
 		log.Label("package_bucket", packagesBucket),
 		log.Label("results_bucket", resultsBucket),
 		log.Label("file_write_results_bucket", fileWritesBucket),
-		log.Label("image_tag", imageTag))
-
+		log.Label("image_tag", imageTag),
+		log.Label("topic_notification", notificationTopicURL))
+	
 	for {
-		err := messageLoop(ctx, subURL, packagesBucket, resultsBucket, fileWritesBucket, imageTag)
-		if err != nil {
+		notificationTopic, err := pubsub.OpenTopic(ctx, notificationTopicURL)
```

Comment:
It might make sense to check if `notificationTopicURL` is empty or not. If so, we should handle that gracefully. We can make `OSSMALWARE_NOTIFICATION_TOPIC` itself optional -- if it's not set, we'll just continue as usual without notifying.

Reply:
@oliverchang would you be able to double check this logic? 
https://github.com/ossf/package-analysis/pull/518/commits/ecc9dd21db29c36fad1e567ecf9580556eb0a330